imagelab_react/src/imagelab-block.js

```diff
@@ -11,7 +11,7 @@ Blockly.defineBlocksWithJsonArray([
         {
           name: "file_browse",
           type: "field_image",
-          src: "imagelab_demo/src/image-plus.png",
+          src: "./image-plus.png",
```

Comment:
why you have to change the path? @abdullahranginwala

Reply:
This path was earlier used for the demo project I had made. When I shifted the project into imagelab and renamed the project to imagelab_react, this path was remained unchanged. It is now rendering the correct image.

Reply:
alright, got it